defs/scenarios/storage/ceph/ceph-osd/PG_overdose.yaml

```diff
@@ -0,0 +1,25 @@
+checks:
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[ne, HEALTH_OK]]
+  # this one will run on OSD sos
```

Comment:
the above comment is misleading since that can run against sos or live host and get the same result